TDP - 23 Add Posterization Shader with Dithering

ASlightlyOvergrownCactus opened the pull request:
Don't worry about the crazy amount of changed files, most of them are just the palette images.

Adds a shader within a test scene that posterizes the entire screen to be a set of colors from a given palette, with dithering from a given bayer matrix. This is meant to be an option for a route of art style that we could take if we want to ^.^, but no worries if not.

ASlightlyOvergrownCactus committed:
Add Posterization Shader with Dithering

Don't worry about the crazy amount of changed files, most of them are just the palette images.

ASlightlyOvergrownCactus committed:
Merge branch 'main' into TDP-13-Posterization-Prototype

funkychicken493 commented on resources/Shaders/TestImages/mainmenu.png:
mainmenu.png needs to be removed we do not have rights to this image

ASlightlyOvergrownCactus replied:
ughhhh booooooring

ASlightlyOvergrownCactus committed:
Remove image cuz funky is BORING

ASlightlyOvergrownCactus committed:
save the thingy

funkychicken493 reviewed (changes requested):
Delete the .godot folder and merge from main

ASlightlyOvergrownCactus committed:
Tweak Dithering to actually be visible

ASlightlyOvergrownCactus committed:
Update import settings

Updated Import settings of all textures to allow for repeating so that shader applies properly.

ASlightlyOvergrownCactus committed:
delete godot folder

ASlightlyOvergrownCactus committed:
Merge branch 'main' into TDP-23-Posterization-Prototype

ASlightlyOvergrownCactus commented:
Ok, .godot folder is fixed and ready to merge

funkychicken493 reviewed (approved):
LGTM

AhmadAJabr reviewed (approved):
Approved
